### PR TITLE
Added sabre to quizPage

### PR DESCRIPTION
**Pull request by PatrykJurczyk**

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -41,8 +41,8 @@ body {
   grid-template-rows: auto 1fr;
```
Comment (FilipHalon) [inline styles/index.css:41]: The saber overlaps with the quiz on tablets and smaller screens. To keep it always below the quiz, you could set `grid-template-rows` to something like `1fr auto 0.2fr` and remove `position: relative` and `top` properties from `#lightSabreProgresBar`. Consider also changing the height of the `img` to make the saber and the time counter closer to the quiz.

**Review by FilipHalon — CHANGES_REQUESTED** [review]

File: src/components/timeSabarCounter/saberTime.js [hunk 2]
```diff
@@ -56,6 +58,12 @@ class Time {
     const bar = document.querySelector('.sabre');
     const leftPercent = Time.countPercent(this.limitTime, this.remainingTime);
     bar.style.width = `${leftPercent}%`;
+    if(leftPercent === 0){
+      // removeElement('#lightSabreProgresBar')
```
Comment (FilipHalon) [inline src/components/timeSabarCounter/saberTime.js:62]: You could remove these comments.

File: styles/index.css [hunk 3]
```diff
@@ -16,7 +16,6 @@ body {
 }
 
 .main-bg-image {
-  display: flex;
```
Comment (FilipHalon) [inline styles/index.css:19]: This should stay, as it makes the content stay centered on wider screens.

File: styles/menu.css [hunk 4]
```diff
@@ -62,7 +62,7 @@
   font-family: Montserrat, sans-serif;
 }
 
-@media screen and (min-width: 320px) {
+@media screen{
```
Comment (FilipHalon) [inline styles/menu.css:65]: I guess you could delete this media query.

**Review by FilipHalon — APPROVED** [review]

**Review by mgoniszewska — APPROVED** [review]

**Pull request merged**